aiosql/queries.py

```diff
@@ -142,9 +142,10 @@ def add_query(self, query_name: str, fn: Callable):
         setattr(self, query_name, fn)
         self._available_queries.add(query_name)
 
-    def add_queries(self, queries: List[Tuple[str, QueryFn]]):
+    def add_queries(self, queries: List[QueryFn]):
```

Comment:
I think I need to undo this interface change, even though it's simpler. It would mean a major version bump I think. I can save this for change for a version 4.